pkg/api/v1alpha1/cluster_webhook_test.go

```diff
@@ -1442,23 +1152,134 @@ func TestClusterUpdateControlPlaneTaintsAndLabelsSuccess(t *testing.T) {
 }
 
 func TestClusterUpdateControlPlaneLabelsInvalid(t *testing.T) {
-	cOld := &v1alpha1.Cluster{
+	cOld := createCluster()
+	cOld.Spec.ControlPlaneConfiguration = v1alpha1.ControlPlaneConfiguration{
+		Labels: map[string]string{
+			"test": "val1",
+		},
+	}
+	cOld.SetManagedBy("management-cluster")
+	c := cOld.DeepCopy()
+	c.Spec.ControlPlaneConfiguration.Labels["test"] = "val1/val2"
+
+	g := NewWithT(t)
+	g.Expect(c.ValidateUpdate(cOld)).NotTo(Succeed())
+}
+
+func TestClusterValidateCreateWorkloadPauseAnnotation(t *testing.T) {
+	features.ClearCache()
+	cOld := createCluster()
+	g := NewWithT(t)
+	cOld.SetManagedBy("management-cluster")
+	cOld.Annotations = map[string]string{}
+	cOld.Annotations[pausedAnnotation] = "true"
+	g.Expect(cOld.ValidateCreate()).To(Succeed())
+}
+
+func TestClusterValidateCreateMgmtPauseAnnotation(t *testing.T) {
+	features.ClearCache()
+	cOld := createCluster()
+	g := NewWithT(t)
+	cOld.SetManagedBy("mgmt")
+	cOld.Annotations = map[string]string{}
+	cOld.Annotations[pausedAnnotation] = "true"
+	g.Expect(cOld.ValidateCreate()).To(Succeed())
+}
+
+func TestClusterValidateCreateActiveFeatureWorkload(t *testing.T) {
+	features.ClearCache()
+	cOld := createCluster()
+	g := NewWithT(t)
+	cOld.SetManagedBy("management-cluster")
+	validateCreateError := cOld.ValidateCreate()
+	g.Expect(validateCreateError).NotTo(Succeed())
+	g.Expect(validateCreateError.Error()).To(ContainSubstring("cluster on existing cluster is not supported"))
```

Comment:
You can do this with one expect

```suggestion
	g.Expect(cOld.ValidateCreate()).To(MatchError(ContainSubstring("cluster on existing cluster is not supported")))
```